H/prototyp.h

```diff
@@ -186,59 +186,7 @@ int csoundSetReleaseLength(void *p, int n);
  */
 MYFLT csoundSetReleaseLengthSeconds(void *p, MYFLT n);
 
-/**
```

Comment:
These functions here look like they were previously a part of the API. Are there alternative ways now to for opcodes to access this information?

Reply:
They don't seem to be used anywhere in the code and they are not public either. I don't know if we should add them to the module API. I tend to try and keep it to what is being used, at this stage I think we should know more or less what opcodes require. Happy to add it if you think it needs to be there.

Reply:
I guess if they weren't public then no worries; if someone comes back that they were using it we'll revisit.